lib/server/push.api.js

```diff
@@ -36,6 +36,15 @@ Push.Configure = function(options) {
     // Add debug info
     if (Push.debug) console.log('Push.Configure', options);
 
+    // config collections for project
+    var prefix = options.dbCollectionPrefix || '';
+    var notificationsCollection =  prefix + '_raix_push_notifications';
+    Push.notifications = new Mongo.Collection(notificationsCollection);
```

Comment:
Why did you moved the collection creation to server only? Shouldn't it be created on client side too?

If your goal is only to add prefix, moving collection to server only side brings this PR to different level.